modules/mirror-registry-remote.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-mirroring-installation-images.adoc
+
+[id="mirror-registry-remote_{context}"]
+= Mirroring on a remote host with mirror registry for {product-title}
+
+Remote mirroring with mirror registry for {product-title}
+
+This procedure introduces how to install Red Hat Quay on a remote host using the `mirror-registry` tool. By doing so, users can create a registry to hold a mirror of {product-title} images.
+
+[NOTE]
+====
+Installing Red Hat Quay using the `mirror-registry` CLI tool makes several changes to your machine. After installation, a `/etc/quay-install` directory is created, which has installation files, local storage, and the configuration bundle. Trusted SSH keys are generated in case the deployment target is the local host, and systemd files on the host machine are set up to ensure that container runtimes are persistent. Additionally, an initial user named `init` is created with an auto-generated password. All access credentials are printed at the end of the install routine.
```

Comment:
```suggestion
Installing the mirror registry using the `mirror-registry` CLI tool makes several changes to your machine. After installation, a `/etc/quay-install` directory is created, which has installation files, local storage, and the configuration bundle. Trusted SSH keys are generated in case the deployment target is the local host, and systemd files on the host machine are set up to ensure that container runtimes are persistent. Additionally, an initial user named `init` is created with an auto-generated password. All access credentials are printed at the end of the install routine.
```